Sources/SwiftSDKGenerator/Generator/SwiftSDKGenerator+Download.swift

```diff
@@ -147,17 +147,12 @@ extension HTTPClient {
   private func downloadUbuntuPackagesList(
     from url: String,
     isVerbose: Bool
-  ) async throws -> String {
-    guard let packages = try await get(url: url).get().body else {
+  ) async throws -> String? {
+    guard var packages = try await get(url: url).get().body?.unzip(isVerbose: isVerbose) else {
       throw FileOperationError.downloadFailed(URL(string: url)!)
     }
 
-    var result = ""
-    for try await chunk in try packages.unzip(isVerbose: isVerbose) {
-      result.append(String(data: chunk, encoding: .utf8)!)
-    }
-
-    return result
+    return packages.readString(length: packages.readableBytes)
```

Comment:
please just `return String(buffer: packages)` and make it a `let packages`

Reply:
see the docs: https://swiftinit.org/docs/swift-nio/niocore/bytebuffer#Basic%20usage

wholesale _conversion_ of ByteBuffer is always `TargetType(buffer: myByteBuffer)` Like `Array(buffer: byteBuffer)`, `String(buffer: myByteBuffer)`, ...